From a0192e7ee765db71956c98c6dc5edb0b5e2e1d8f Mon Sep 17 00:00:00 2001
From: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Date: Sun, 25 Mar 2018 08:41:52 -0500
Subject: [PATCH 3543/5242] MLK-17703-10: drm: imx: dcss: remove unused
 variable warning

commit  15e28b81fccb0ab2ad550929bcaa81761a9c45d6 from
https://source.codeaurora.org/external/imx/linux-imx.git

dcss_crtc variable was not used anymore and generated a compilation
warning. Remove it.

Signed-off-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/dcss/dcss-crtc.c |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/imx/dcss/dcss-crtc.c b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
index 5e80bc2..d02ab5a 100644
--- a/drivers/gpu/drm/imx/dcss/dcss-crtc.c
+++ b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
@@ -137,9 +137,6 @@ static int dcss_crtc_atomic_check(struct drm_crtc *crtc,
 static void dcss_crtc_atomic_begin(struct drm_crtc *crtc,
 				   struct drm_crtc_state *old_crtc_state)
 {
-	struct dcss_crtc *dcss_crtc = container_of(crtc, struct dcss_crtc,
-						   base);
-
 	drm_crtc_vblank_on(crtc);
 
 	spin_lock_irq(&crtc->dev->event_lock);
-- 
1.7.9.5

